jsk_robot_common/jsk_robot_startup/util/mux_selector.py

```diff
@@ -51,12 +51,11 @@ def add_trigger(topic, expr, select, index):
     topic_class, _, _ = rostopic.get_topic_class(topic)
 
     if(topic_type is None):
-        rospy.loginfo('%s is not published yet', topic)
-        return None
+        rospy.logwarn('%s is not published yet', topic)
 
     if(topic_class is None):
-        rospy.loginfo('%s is not builded yet', topic_type)
-        return None
+        rospy.logwarn('%s is not builded yet', topic_type)
+        topic_class = rospy.AnyMsg
```

Comment:
Please explain the purpose of this line.

Reply:
Thank you for your review. 

Without this pull request,
1. I want to use `topic_tools/relay` to use the same topic as the system diagram
2. The type of `topic_tools/relay` output topic is not defined when the teleop system starts.
3. `mux_selector.py` cannot subscribe the output topic
https://github.com/jsk-ros-pkg/jsk_robot/blob/b7163007df321d3e9349cbfe1bb471bb163056fd/jsk_robot_common/jsk_robot_startup/util/mux_selector.py#L64

Therefore, I use `rospy.AnyMsg` if the topic type is not defined.

Reply:
I see.
Did you see the https://github.com/jsk-ros-pkg/jsk_robot/pull/1195 ?
You can cope with the problem with its `wait` option.

Reply:
Thank you for your advice.
I did not know https://github.com/jsk-ros-pkg/jsk_robot/pull/1195, but now I think https://github.com/jsk-ros-pkg/jsk_robot/pull/1195 is useful for this case.

I pushed additional commits.